### PR TITLE
Minor bug fixes to HMRC manual layout and breadcrumbs 

### PR DESCRIPTION
**Pull request by hannako**
https://trello.com/c/DwMxfbuf/1145-breadcrumb-issues-in-hmrc-internal-manuals

#### 1. Don't display the "Summary" heading if there's no content below it.

Eg on https://www.gov.uk/hmrc-internal-manuals/capital-gains-manual 

|Before|After|
|-|-|
|Screenshot 2024-07-29 at 22 58 32|Screenshot 2024-07-29 at 22 57 53|

Here's a page that has a populated description [on live](https://www.gov.uk/hmrc-internal-manuals/debt-management-and-banking) and [on the review app](https://government-frontend-pr-3284.herokuapp.com//hmrc-internal-manuals/debt-management-and-banking) to show that there's no visible change.

#### 2. If breadcrumbs are present, display them

Eg debt management and banking manual
- [missing on live](https://www.gov.uk/hmrc-internal-manuals/debt-management-and-banking/dmbm510110)
- [present on review app](https://government-frontend-pr-3284.herokuapp.com/hmrc-internal-manuals/debt-management-and-banking/dmbm510110)

Eg capital gains manual
- [missing on live](https://www.gov.uk/hmrc-internal-manuals/capital-gains-manual/cg40432)
- [present on review app]( https://government-frontend-pr-3284.herokuapp.com/hmrc-internal-manuals/capital-gains-manual/cg40432)

|Before|After|
|-|-|
|Screenshot 2024-07-29 at 23 00 16|Screenshot 2024-07-29 at 23 01 51|

Follow [these steps](https://guides.rubyonrails.org/upgrading_ruby_on_rails.html) if you are doing a Rails upgrade.

### CODE REVIEW

File: app/views/content_items/manuals/_manual_layout.html.erb [hunk 1]
```diff
@@ -1,8 +1,10 @@
...
-  <%= render "govuk_publishing_components/components/heading", {
-    text: t("manuals.summary_title"),
-    margin_bottom: 4
-  } %>
+  <% if @content_item.description.present? %>
+    <%= render "govuk_publishing_components/components/heading", {
+      text: t("manuals.summary_title"),
+      margin_bottom: 4
+    } %>
+  <% end %>
```
Comment (KludgeKML) [inline app/views/content_items/manuals/_manual_layout.html.erb:7]: Not a blocking comment but: one thing that seems odd about this section is that the summary title and the text of the summary are separated in such a way that they need two separate if statements testing whether the description is present. It feels a bit like if the description is inside the body, the summary title ought to be too. But probably an FE dev would have to make that call. As things stand, this code change is fine.

**Review by KludgeKML — APPROVED** [review]
LGTM (one non-blocking comment)

File: app/views/content_items/manuals/_manual_layout.html.erb (hunk 1 above)
Comment (hannako) [reply app/views/content_items/manuals/_manual_layout.html.erb:7]: Oh yeah good point. Let me do that and see if it looks OK and get a FE dev to 👀

**Review by hannalaakso — APPROVED** [review]
The first two commits LGTM 👍

**Pull request merged**